src/basket/BasketMiniView.tsx

```diff
@@ -195,15 +195,14 @@ const BasketMiniViewTab = ({
             ? `Remove (${selectedEntries.length})`
             : `Clear all (${accessions.length})`}
         </Button>
-        <Button
-          element={Link}
-          variant="secondary"
+        <Link
```

Comment:
Was getting too complicated to support types for both `button` and `Link` within franklin's `Button`.
So now `Button` is for underlying `button` and a `Link` that should look like a button will have to directly use classes `button` and `<variant>` from Franklin